src/java.base/share/classes/java/time/Duration.java

```diff
@@ -1419,7 +1419,8 @@ public Duration truncatedTo(TemporalUnit unit) {
      * It is "consistent with equals", as defined by {@link Comparable}.
      *
      * @param otherDuration the other duration to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value is less than zero if the other duration is before,
```

Comment:
General comment.  I am not sure that "other duration" is quite clear.  It seems that javadoc for many `compareTo` methods have not standardized on wording for describing the argument being checked.   Some include "argument" when the documentation.    Perhaps change "other duration" -> "duration argument"?

I think LocalDate::compareTo and its use of "other" is perhaps a better  example of why we might want to beef the wording up a bit

Reply:
The argument name is `otherDuration` would putting the argument name in {@code} font put sufficient focus on the argument?

Reply:
I think that would be fine as well as then it is clear we are referring to the argument.